problems/objects.js

```diff
@@ -187,15 +264,54 @@ const countOccurance = () => {};
  * @returns {Object} Counts all characters except spaces  {a: 2, g: 1, o: 2, d:1, s: 1, n:a, k:1, e: 1}
  */
 
-const countOccuranceNoSpaces = () => {};
+const countOccuranceNoSpaces = (str) => {
+  let stringArr = str.toLowerCase().split(" ");
+  let joinArr = stringArr.join("");
+  let countObject = {}
+  // console.log(stringArr)
+  // console.log(joinArr)
+  for(let i = 0; i < joinArr.length; i++) {
+    let stringNoSpace = joinArr[i]
+    if(countObject[stringNoSpace]) {
+      countObject[stringNoSpace] += 1;
+    } else {
+      countObject[stringNoSpace] = 1
+    }
+  }
+  return countObject
+};
+// console.log(countOccuranceNoSpaces("HELLO world"))
+
 
 /**
  * Takes in an array and returns the most common element.
  * @param {Array} array - array of elements
  * @returns {(number|string)} Most common element
  */
 
-const mostCommonElement = () => {};
+const mostCommonElement = (array) => {
```

Comment:
How do you think we could use your countOccurence function to solve this question?